ksqldb-common/src/main/java/io/confluent/ksql/config/ImmutableProperties.java

```diff
@@ -34,6 +34,7 @@
       .add(KsqlConfig.KSQL_HIDDEN_TOPICS_CONFIG)
       .add(KsqlConfig.KSQL_READONLY_TOPICS_CONFIG)
       .add(KsqlConfig.KSQL_SOURCE_TABLE_MATERIALIZATION_ENABLED)
+      .add(KsqlConfig.KSQL_HEADERS_COLUMNS_ENABLED)
```

Comment:
So it can't be enabled from the client side. Otherwise, ksql admins won't have control on what to enable/disable 'cause clients can be enabled it from the UI or CLI.